src/main/java/org/kainos/ea/api/DeliveryService.java

```diff
@@ -1,2 +1,42 @@
-package org.kainos.ea.api;public class DeliveryService {
+package org.kainos.ea.api;
+
+import org.kainos.ea.cli.DeliveryEmployee;
+import org.kainos.ea.cli.DeliveryRequest;
+import org.kainos.ea.client.DeliveryEmployeeDoesNotExistException;
+import org.kainos.ea.client.FailedToCreateDeliveryEmployeeeException;
+import org.kainos.ea.db.DeliveryDao;
+
+import java.sql.SQLException;
+
+public class DeliveryService {
+    private DeliveryDao deliveryDao = new DeliveryDao();
+
+    public int createDeliveryEmployee(DeliveryRequest deliveryRequest) throws FailedToCreateDeliveryEmployeeeException {
+        try {
+            int id = deliveryDao.createDeliveryEmployee(deliveryRequest);
+
+            if (id == -1) {
+                throw new FailedToCreateDeliveryEmployeeeException();
+            }
+            return id;
+        } catch (SQLException e) {
+            System.err.println(e.getMessage());
+        }
+        return -1;
```

Comment:
If we fail to create due to a SQL error currently we would log a message and just return -1 to the controller which would return a 200 response